src/test/ui/unboxed-closures/unboxed-closure-sugar-wrong-trait.stderr

```diff
@@ -1,8 +1,8 @@
 error[E0244]: wrong number of type arguments: expected 0, found 1
-  --> $DIR/unboxed-closure-sugar-wrong-trait.rs:15:8
+  --> $DIR/unboxed-closure-sugar-wrong-trait.rs:15:13
    |
 LL | fn f<F:Trait(isize) -> isize>(x: F) {}
-   |        ^^^^^^^^^^^^^^^^^^^^^ expected no type arguments
+   |             ^^^^^^^^^^^^^^^^ unexpected type argument
```

Comment:
Hmm this span seems wrong, it should end at the `)`, I wonder if that's a bug in how `T(A) -> R` turns into `T<(A,), Output = R>`.

Reply:
Yeah, this one looked a little weird to me, but I thought it was probably due to function syntax issues that weren't related to this change.

Reply:
Open an issue so we can maybe clean this up in the future.

Reply:
Opened https://github.com/rust-lang/rust/issues/53534.